playbooks/troubleshooting/troubleshooting_guide.adoc

```diff
@@ -0,0 +1,724 @@
+---
+---
+=  Troubleshooting Guide for OpenShift Enterprise 3.1
+
+Steve Ovens
+ +
+Justin Jarae
+
+v1.0, 2016-03-11
+
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+toc::[]
```

Comment:
Your TOC does not render when I build the site. This may have to do with the author header being misformatted.